src/zen/websocket_server.cpp

```diff
@@ -338,15 +338,20 @@ class WsHandler
             }
             if (pblockindex == NULL)
             {
-                LogPrint("ws", "%s():%d - block index not found for hash[%s]\n", __func__, __LINE__, o.get_str());
-                return INVALID_PARAMETER;
+                LogPrint("ws", "%s():%d - block index not found for hash[%s], skipping it\n", __func__, __LINE__, o.get_str());
+                continue;
             }
             if (pblockindex->nHeight > lastH)
             {
                 lastH = pblockindex->nHeight;
                 pblockindexStart = pblockindex;
             }
         }
+        if (pblockindexStart == NULL)
+        {
+            LogPrint("ws", "%s():%d - could not find start index\n", __func__, __LINE__);
+            return INVALID_PARAMETER;
+        }
         listBlock.push_back(pblockindexStart);
```

Comment:
@alsala I think we have one more error here.
Let's assume that in `hashes` we also retrieved current Tip hash. So current Tip will we set as `pblockindexStart`.
In next lines 355-363 we will always respond with `INVALID_PARAMETER` error:
```
            CBlockIndex* nextBlock = chainActive.Next(pblockindexStart);
            if (nextBlock == NULL)
            {
                LogPrint("ws", "%s():%d - next block index not found\n", __func__, __LINE__);
                return INVALID_PARAMETER;
            }
```